docs/aws/services/ebs.md

```diff
@@ -0,0 +1,23 @@
+# Amazon Elastic Block Store (Amazon EBS)
+
+## Service Details
+
+Amazon Elastic Block Store (Amazon EBS) provides block level storage volumes for use with EC2 instances.
+
+## Assessment Notes
+* Encryption at rest should be enabled
+  * Default encryption should be enabled for each region at the account level. This can only be checked through the console - [https://aws.amazon.com/blogs/aws/new-opt-in-to-default-encryption-for-new-ebs-volumes/](https://aws.amazon.com/blogs/aws/new-opt-in-to-default-encryption-for-new-ebs-volumes/)
+  * In an ideal world, EBS volumes should be encrypted with CMKs with proper access controls configured in the key policies. This is, however, exceptionally low priority - [https://www.chrisfarris.com/post/cloud-encryption/](https://www.chrisfarris.com/post/cloud-encryption/)
+* Snapshots should not be public
```

Comment:
A reference here to a few examples on public snapshot damage:
https://techcrunch.com/2019/08/09/aws-ebs-cloud-backups-leak/
https://defcon.org/html/defcon-27/dc-27-speakers.html#Morris + slides: https://media.defcon.org/DEF%20CON%2027/DEF%20CON%2027%20presentations/DEFCON-27-Ben-Morris-More-Keys-Than-A-Piano-Finding-Secrets-In-Publicly-Exposed-Ebs-Volumes.pdf